packages/application/src/shell.ts

```diff
@@ -338,6 +338,31 @@ export class LabShell extends Widget implements JupyterFrontEnd.IShell {
       rootLayout.insertWidget(2, this._menuHandler.panel);
     }
 
+    // Skip Links
+    const skipLinkWidget = (this._skipLinkWidget = new Widget());
+    const skipToFilterFiles = document.createElement('a');
+    const toggleSkipLink = (toShow: boolean) => {
+      skipLinkWidget.node.classList.toggle('jp-skiplink--focus', toShow);
+    };
```

Comment:
wrap it up into a function

Reply:
^^ Is that something you're planning to do?

Reply:
Oh...yes! I forgot. 🤦‍♀️